rt/rs/description/src/main/java/org/apache/cxf/jaxrs/model/wadl/WadlGenerator.java

```diff
@@ -32,6 +32,7 @@
 import java.lang.reflect.TypeVariable;
 import java.net.URI;
 import java.net.URL;
+import java.nio.charset.StandardCharsets;
 import java.util.ArrayList;
```

Comment:
Please remove the ByteArrayOutputStream import which is no longer required, to fix a checkstyle problem.

Reply:
Done.